pkg/services/sqlstore/sqlstore.go

```diff
@@ -568,29 +578,6 @@ func TestMain(m *testing.M) {
 
 		testSQLStoreCleanup = append(testSQLStoreCleanup, testDB.Cleanup)
 
-		// useful if you already have a database that you want to use for tests.
```

Comment:
Doesn't removing this break existing tests? 🤔

Reply:
Not in OSS, but I hadn't thought to check Enterprise, where it does indeed break tests. 😢